code/game/objects/items/stacks/medical.dm

```diff
@@ -18,6 +18,8 @@
 	var/other_delay = 0
 	var/repeating = FALSE
 	var/experience_given = 1
+	/// Shiptest Edit - Sound/Sounds to play when this is applied
```

Comment:
We don't do this anymore.

Reply:
Are you talking about the Shiptest Edit or autodocing?

Reply:
Yes